.circleci/config.yml

```diff
@@ -155,7 +155,7 @@ jobs:
                     -kubecontext="kind-dc1" \
                     -secondary-kubecontext="kind-dc2" \
                     -debug-directory="$TEST_RESULTS/debug" \
-                    -consul-k8s-image=docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest
+                    -consul-k8s-image=ghcr.io/lkysow/consul-k8s-dev:feb09
```

Comment:
I wasn't able to test using your ghcr image or the one created by consul-k8s-ci because I was seeing this error in both cases:
```
k logs consul-consul-connect-injector-webhook-deployment-7f69bc4dftckh
flag provided but not defined: -cleanup-controller-reconcile-period
```
I'll code review the rest for now.